_drafts/2023-02-13-cheating-the-system-for-fun-and-profit.md

```diff
@@ -0,0 +1,129 @@
+---
+layout: post
+permalink: cheating-the-system-for-fun-and-profit
+title: 'Cheating the System for Fun and Profit'
+tags: [accessibility]
+---
+
+Okay, that title might just be a little bit of clickbait. The idea is to use
+macOS VoiceOver and Hover Text to navigate through Xcode's simulator, helping
+test your (iOS) app without having to take out your iOS device... mostly. But
+we'll get to that.
+
+<!--more-->
+
+## Hover Text
+
+Hover Text is a macOS feature that let's you view text at larger sizes,
+typically using a modifier key and hovering over the text or element (hence the
+name). What's neat about it, is that is will show you the accessibility label
+of any element, not just text, including in the iOS simulator. Meaning Hover
+Text can be a rather quick and frictionless way to get an idea of elements not
+having an accessibility label, or having an awkward one. You can find Hover Text
+under System Settings (or System Preferences if you're not yet running macOS 13
+Ventura) > Accessibility > Zoom > Hover Text.
+
+![Hover Text in the macOS system settings.](./assets/blog-assets/(hover-text.png)
+
+And this is what Hover Text looks like in practice, here as seen in the
+simulator.
+
+![Hover Text in the macOS system settings.](./assets/blog-assets/(hover-text-in-simulator.png)
```

Comment:
<img width="457" alt="hover-text-in-simulator" src="https://user-images.githubusercontent.com/4190298/218311615-81861b88-199f-4462-8b74-abdd6aa2fd08.png">